ArduPlane/quadplane.cpp

```diff
@@ -3175,14 +3180,6 @@ bool QuadPlane::guided_mode_enabled(void)
     return guided_mode != 0;
 }
 
-/*
-  set altitude target to current altitude
- */
-void QuadPlane::set_alt_target_current(void)
-{
-    pos_control->set_alt_target(inertial_nav.get_altitude());
```

Comment:
keep this, but use set_alt_target_to_current_alt

Reply:
removed this change